muni_portal/tests/api/test_user_account.py

```diff
@@ -0,0 +1,91 @@
+from django.contrib.auth.models import User
+from django.urls import reverse
+from faker import Faker
+from rest_framework import status
+from rest_framework.test import APITestCase
+from unittest.mock import patch
+from urllib.parse import urlparse, parse_qsl
+
+
+class ApiUserAccountTestCase(APITestCase):
+
+    def setUp(self):
+        self.password = Faker().password(length=8)
+
+    @patch("rest_registration.notifications.email.send_notification")
```

Comment:
please test email using https://docs.djangoproject.com/en/2.2/topics/testing/tools/#email-services instead of custom mocks

Reply:
In this way, I should parse the entire email for the verification link and I'm not testing emails in this test.
Patching is default approach for testing those things and I didn't use provided services at all.

Reply:
This is intended to be an integration test - we're not trying to test whether the libraries do what they promise, we're trying to test whether we're using the libraries in a way that can actually be used by an end user. A regex something like `(https?://example.com/confirm-email/)` and then parsing the query string is totally fine, and in line with what we're trying to do.